app/src/routes/useChoiceBoardModel.ts

```diff
@@ -1,15 +1,39 @@
-import { useState } from 'react';
+import { useState,useEffect} from 'react';
 
 export default () => {
   // Selected Items
   const [selectedItems, setSelectedItems] = useState([]);
 
   // Initial choices
-  const initialOptions = [];
-  // useEffect(async () => {
-  //  const response = await fetch(process.env.REACT_APP_BACKEND_URL);
-  //  initialOptions = [response.body.choices[0], response.body.choices[1]];
-  // })
+  
+  /*useEffect(async () => {
+   const response = await fetch(process.env.REACT_APP_BACKEND_URL);
+    initialOptions = [response.body.choices[0], response.body.choices[1]];
+   })*/
+   
+  //fetching the random user details for initial choices 
+let url = "https://randomuser.me/api/?results=5";
```

Comment:
Please, change this URL to use the **mock-server** because the JSON used must follow the following format https://github.com/bancodobrasil/stop-analyzing-api/issues/29#issuecomment-702963899

Reply:
sure